### DISCUSSION TITLE
Dress LORA only creating the same pose as in the training data

### DISCUSSION THREAD

**Discussion by nitinh12** [bmaltais/kohya_ss#2707] [Q&A]
I trained a LORA for a dress and while it creates great images but all the time creates the same pose as in the training images. I even captioned the pose and used it in the negative prompt and even added weight to it but still, it creates the same pose. I trained it on SDXL base model and I am using it in Fooocus. Not sure what I am doing wrong. Please help me 🙏

**Comment by kalle07** [comment]
u should use at least 5 poses with at least 3 images with same pose so 15 images ( you can flip the images) ...
check first how the checkpoint u use react on the poses you want and if it support more ...
so eg if you dont train a pose from behind it is hard to get a pose from behinde...
